test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/TestSuitInitializer.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common;
+
+/* Responsible for all initilisation before actual test startup -
+* Creation of Azure resources - Eventhubs and Storage Queues
+* Function App startup
+*/
+public class TestSuitInitializer
+{
```

Comment:
Can you review the exception handling in this and called classes. I see this is interacting with 3rd party services and it should have a proper exception handling, else everything will be bubbled up as unhandled exceptions which wil be hard to debug and diagnose.